### DISCUSSION TITLE
toml_edit: deserialize an `Item`

### DISCUSSION THREAD

**Discussion by banditopazzo** [toml-rs/toml#611] [Q&A]
In a previous test with an old version (`0.15.0`) I was able to deserialize an `Item` into a `T`, but in the latest version I can't find a way to do the same thing, is it still possible?

**Comment by epage** [comment]
If you can convert it into a `Document`, you can use [Deserializer](https://docs.rs/toml_edit/latest/toml_edit/de/struct.Deserializer.html)

If it can be converted to `Value`, you can use [ValueDeserializer](https://docs.rs/toml_edit/latest/toml_edit/de/struct.ValueDeserializer.html)
